### PR TITLE
Update dutch

### LINKED ISSUE #65: Change in translation (Dutch)
**Is your feature request related to a problem? Please describe.**
It is not related to a problem but to a translation (Dutch)

**Describe the solution you'd like**
The dutch translation states "Waarschuwing geel mist" which if you read it indicates that we should be warned for yellow mist.
Same with the warning for red. In The Netherlands we use "Code Yellow of Code Red"

**Additional context**
Change the text to:

Geen Waarschuwingen
Code Geel:
Code Rood:

### PR DESCRIPTION
**Pull request by BartEngelen**
Fixes #65

### CODE REVIEW

**Review by MrBartusek — CHANGES_REQUESTED** [review]
Hey, Thanks for the PR!

However, your `master` branch is outdated and the translation files have been updated a couple of times since then. Please do rebase your `master` and work on the new files.

**Comment by BartEngelen** [comment]
Hello,
Can you help me on this one ?
Kind regards,
Bart Engelen

**Comment by MrBartusek** [comment]
You can try using this button on your fork and then rebasing once again 
image

**Comment by BartEngelen** [comment]
Hello,
I got this error:
image
What can I do to fix this ?
Regards,
Bart Engelen

**Comment by MrBartusek** [comment]
You are still working on an old version of master

**Comment by MrBartusek** [comment]
There hasn't been any activity in this pull request recently so I will close it. Feel free to create a new one if you want to continue your work. Thanks for your contribution!

File: src/translations/nl.json [hunk 1]
```diff
@@ -34,8 +39,20 @@
     "rain_flood": "wateroverlast"
   },
   "messages": {
-    "yellow": "Waarschuwing geel {0}",
```
Comment (MDLefevere) [inline src/translations/nl.json:37]: "Code Geel: {0}"

File: src/translations/nl.json [hunk 2]
```diff
@@ -34,8 +39,20 @@
...
-    "orange": "Waarschuwing oranje {0}",
```
Comment (MDLefevere) [inline src/translations/nl.json:38]: "Code Oranje: {0}"

File: src/translations/nl.json [hunk 3]
```diff
@@ -34,8 +39,20 @@
...
-    "red": "Waarschuwing rood {0}"
```
Comment (MDLefevere) [inline src/translations/nl.json:39]: "Code Rood: {0}"

**Review by MDLefevere — COMMENTED** [review]
These 3 lines need to be changed for the right text.

**Pull request closed without merging**